it/mail/chrome/messenger/addressbook/ldapAutoCompErrs.properties

```diff
@@ -91,48 +91,48 @@ errorAlertFormat=Codice di errore %1$S: %2$S\n\n %3$S
 
 ## @name STRONGAUTH_REQUIRED_HINT
 ## @loc none
-10008=L'autenticazione forte non è ancora supportata.
+10008=L'autenticazione a più fattori non è attualmente supportata.
 
 ## @name INVALID_SYNTAX_HINT
 ## @loc none
-10021=Verificare che il filtro di ricerca sia corretto quindi riprovare o contattare l'Amministratore di Sistema. Per verificare la correttezza del filtro dal menu Modifica scegliere Preferenze, quindi Posta e gruppi di discussione ed infine Indirizzamento. Fare clic su Modifica Directory e selezionare il server LDAP che si sta usando. Infine fare clic su Modifica e quindi su Avanzate per mostrare il filtro di ricerca.
+10021=Verifica che il filtro di ricerca sia corretto e poi riprova oppure contatta il tuo Amministratore di sistema. Per verificare che il filtro sia corretto, scegli Preferenze dal menu Modifica, poi E-mail e Newsgroup e poi Indirizzamento. Clicca su Modifica directory e seleziona il server LDAP che stai utilizzandoo. Infine clicca su Modifica e poi su Avanzate per mostrare il Filtro di ricerca.
 
 ## @name NO_SUCH_OBJECT_HINT
 ## @loc none
-10032=Verificare che il Base DN sia corretto quindi riprovare o contattare l'Amministratore di Sistema. Per verificare la correttezza del Base DN dal menu Modifica scegliere Preferenze, quindi Posta e gruppi di discussione ed infine Indirizzamento. Fare clic su Modifica Directory e selezionare il server LDAP che si sta usando. Infine fare clic su Modifica per mostrare il Base DN.
+10032=Verifica che Base DN sia corretto e poi riprova oppure contatta il tuo Amministratore di sistema. Per verificare che Base DN sia corretto, scegli Preferenze dal menu Modifica, poi E-mail e Newsgroup e poi Indirizzamento. Clicca su Modifica directory e seleziona il server LDAP che stai utilizzando. Infine clicca su Modifica per mostrare Base DN.
 
 ## @name BUSY_HINT
 ## @loc none
-10051=Riprovare più tardi.
+10051=Riprova più tardi.
 
 ## @name SERVER_DOWN_HINT
 ## @loc none
-10081=Verificare che il nome dell'host remoto e la porta siano corretti quindi riprovare o contattare l'Amministratore di Sistema. Per verificare la correttezza del nome dell'host remoto e della porta dal menu Modifica scegliere Preferenze, quindi Posta e gruppi di discussione ed infine Indirizzamento. Fare clic su Modifica Directory e selezionare il server LDAP che si sta usando. Infine fare clic su Modifica per visualizzare il nome dell'host remoto e Avanzate per visualizzare la porta.
+10081=Verifica che il nome dell'host e il numero della porta siano corretti e poi riprova oppure contatta il tuo Amministratore di sistema. Per verificare che il nome dell'host e il numero della porta siano corretti, scegli Preferenze dal menu Modifica, poi Posta e Newsgroup e poi Indirizzamento. Clicca su Modifica directory e seleziona il server LDAP che stai utilizzando. Clicca su Modifica per visualizzare il nome dell'host Clicca su Avanzate per visualizzare il numero della porta.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

 +10081=Verifica che il nome dell'host e il numero della porta siano corretti e poi riprova oppure contatta l'Amministratore di sistema. Per verificare che il nome dell'host e il numero della porta siano corretti, scegli Preferenze dal menu Modifica, poi Posta e Newsgroup e poi Indirizzamento. Clicca su Modifica directory e seleziona il server LDAP che stai utilizzando. Clicca su Modifica per visualizzare il nome dell'host Clicca su Avanzate per visualizzare il numero della porta.